test/Particle.spec.js

```diff
@@ -2021,6 +2021,22 @@ describe('ParticleAPI', () => {
 				});
 			});
 		});
+
+		describe('.deleteUser', () => {
+			it('sends request to delete the current user', () => {
+				return api.deleteUser(props).then(result => {
+					// eslint-disable-next-line no-console
+					console.log(result);
```

Comment:
sorry to be a pain but no `console` please :pray:

Reply:
Oops—thanks for catching that! 🧹